src/component/mxgraph/shape/task-shapes.ts

```diff
@@ -176,24 +176,84 @@ export class UserTaskShape extends BaseTaskShape {
   // use mxgraph svg2xml to generate the xml stencil and port it to code
   protected paintTaskIcon(c: mxgraph.mxXmlCanvas2D, x: number, y: number, w: number, h: number): void {
     // icon coordinates fill a 12x13 rectangle
```

Comment:
:warning: comment is out of date, the rectangle is now 239x???
Such comment will be removed when implementing #247